javascriptv3/example_code/cloudwatch-events/actions/put-rule.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// snippet-start:[javascript.v3.cloudwatchlogs.actions.PutRule]
+import { PutRuleCommand } from "@aws-sdk/client-cloudwatch-events";
+import { client } from "../libs/client.js";
+
+const run = async () => {
+  const command = new PutRuleCommand({
+    /**
```

Comment:
Does this align with your style in other examples?

Reply:
Not particularly, no. I'll update.

Reply:
On second thought, previous patterns I've used are not as flexible. I'm going to shift towards this.